x-pack/filebeat/docs/inputs/input-http-endpoint.asciidoc

```diff
@@ -255,6 +268,10 @@ For example, `["content-type"]` will become `["Content-Type"]` when the filebeat
 
 This option copies the raw unmodified body of the incoming request to the event.original field as a string before sending the event to Elasticsearch.
 
+[float]
+==== `crc.provider`
+
+This option defines the provider of the webhook that uses CRC (Challenge-Response Check) for validating the endpoint. The HTTP endpoint input is responsible for ensuring the authenticity of incoming webhook requests by generating and verifying a unique token. By specifying the `crc.provider`, you ensure that the system correctly handles the specific CRC validation process required by the chosen provider.
```

Comment:
Also, the doc does not mention that `crc.provider` should be strictly lower case

Reply:
This is not the case, the providers are lower-cased at map look-up, [here](https://github.com/elastic/beats/pull/35204/files#diff-72430a3a126776d3059fc756070316d0c7a70e799c250cbac4193dd6126c4489R104) and [here](https://github.com/elastic/beats/pull/35204/files#diff-39de92badeeb1ad6d0b17c175de3f81d278937d17bef91f2cc840d97114535ebR40).